ParameterDataApi.yaml

```
@@ -143,60 +199,69 @@ components:
     HeaderDefinition:
       type: object
       properties: 
-        startTime:
-          description: The start time for the time series
-          type: string
-          format: date-time
-        
-        endTime: 
-          description: The end time for the time series
+        messageType:
+          type: string          
+          description: The type of the message.
+
+        version:
```

Not the same API but consistency with the recent header change to be explicit.
```suggestion
        messageVersion:
```